godtossing/Pos/Ssc/GodTossing/Workers.hs

```diff
@@ -121,7 +120,7 @@ shouldParticipate epoch = do
     ourId <- getOurStakeholderId
     let enoughStake = ourId `HM.member` richmen
     when (participationEnabled && not enoughStake) $
-        logDebug "Not enough stake to participate in MPC"
+        logDebugS "Not enough stake to participate in MPC"
```

Comment:
Seems to be redundant, since there are lots of users that don't pass MPC threshold.

Reply:
Makes sense indeed.

Reply:
> since there are lots of users that don't pass MPC threshold

if you don't make this message sensitive, then its absence indicates that you pass the MPC threshold